backend/uclapi/common/cachet.py

```diff
@@ -0,0 +1,51 @@
+import cachetclient.cachet as cachet
+import json
+from django.conf import settings
+
+ENDPOINT = settings.CACHET_URL
+API_TOKEN = settings.CACHET_TOKEN
+
+def create_incident(error_message, component_name):
+    try:
+        components = cachet.Components(endpoint=ENDPOINT, api_token=API_TOKEN)
+        comp_list = json.loads(components.get())
+        target_comp = None
+        for i in comp_list["data"]:
+            if i["name"] == component_name:
+                target_comp = i
+                break
+        if target_comp["status"] == 1:
+            incidents = cachet.Incidents(endpoint=ENDPOINT, api_token=API_TOKEN)
+            message = f"Gencache failed on "
+                      f"{settings.UCLAPI_DOMAIN_CURRENT}"
+                      f" with error: {repr(error_message)}"
+            new_incident = json.loads(incidents.post(name='Gencache failed',
+                                 message=message,
+                                 status=1,
+                                 component_id=i["id"],
+                                 component_status=4))
```

Comment:
```suggestion
            new_incident = json.loads(
                incidents.post(
                    name='Gencache failed',
                    message=message,
                    status=1,
                    component_id=i["id"],
                    component_status=4
                )
            )
```
Hound would help here I imagine. But in any case, my confusion here is the indentation of the inner-most arguments (after the first) seem off.